packages/components/src/Components/FormElements/InputRadio/InputRadio.js

```diff
@@ -0,0 +1,188 @@
+      border-color: ${formColors.colorInputBorderFocus};
+      box-shadow: inset 0 0 0 1px ${formColors.colorInputBorderFocus};
+    }
+    &:checked:focus {
+      box-shadow: 0 0 0 1px ${formColors.colorInputBg},
+        0 0 0 4px ${formColors.colorInputFocusShadow},
+        inset 0 0 0 1px ${formColors.colorInputFg};
+    }
+    &:checked:hover {
+      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' height='10' width='10'%3E%3Ccircle cx='5' cy='5' r='5' fill='%23222330' /%3E%3C/svg%3E");
+      border-color: ${formColors.colorInputFg};
+    }
+  `,
+  radioWithError: css`
+    border-color: ${formColors.colorInputBorderError};
+    box-shadow: inset 0 0 0 1px ${formColors.colorInputBorderError};
+    &:hover {
+      border-color: ${formColors.colorInputFg};
+      box-shadow: inset 0 0 0 1px ${formColors.colorInputFg};
+    }
+    &:focus {
+      box-shadow: 0 0 0 1px ${formColors.colorInputBg},
+        0 0 0 4px ${formColors.colorInputFocusShadow},
+        inset 0 0 0 1px ${formColors.colorInputBorderError};
+    }
+    &:focus:hover {
+      border-color: ${formColors.colorInputFg};
+      box-shadow: 0 0 0 1px ${formColors.colorInputBg},
+        0 0 0 4px ${formColors.colorInputFocusShadow},
+        inset 0 0 0 1px ${formColors.colorInputFg};
+    }
+    &:checked {
+      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' height='10' width='10'%3E%3Ccircle cx='5' cy='5' r='5' fill='%23D72222' /%3E%3C/svg%3E");
+      border-color: ${formColors.colorInputBorderError};
+      box-shadow: inset 0 0 0 1px ${formColors.colorInputBorderError};
+    }
+    &:checked:focus {
+      border-color: ${formColors.colorInputBorderError};
+      box-shadow: 0 0 0 1px ${formColors.colorInputBg},
+        0 0 0 4px ${formColors.colorInputFocusShadow},
+        inset 0 0 0 1px ${formColors.colorInputBorderError};
+    }
+    &:checked:hover {
+      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' height='10' width='10'%3E%3Ccircle cx='5' cy='5' r='5' fill='%23222330' /%3E%3C/svg%3E");
+      box-shadow: inset 0 0 0 1px ${formColors.colorInputFg};
+      border-color: ${formColors.colorInputFg};
+    }
+    &:checked:focus:hover {
+      box-shadow: 0 0 0 1px ${formColors.colorInputBg},
+        0 0 0 4px ${formColors.colorInputFocusShadow},
+        inset 0 0 0 1px ${formColors.colorInputFg};
+      border-color: ${formColors.colorInputFg};
+    }
+  `,
+  label: css`
+    color: ${colors.text};
+    cursor: pointer;
+  `,
+  labelRequired: css`
+    :after {
+      content: ' *';
+      color: ${colors.maximumRed};
+    }
+  `,
+};
+
+/**
+ * Renders an Input type radio.
+ *
+ * Example:
+ *
+ * @code
+ * <label htmlFor="Radio" required={true}><input type="radio">Radio</input></label>
```

Comment:
Don't you want to use <InputRadio \> in the example here?

Reply:
fixed